src/controller/CHIPDevice.cpp

```diff
@@ -175,8 +175,19 @@ CHIP_ERROR Device::Deserialize(const SerializedDevice & input)
     return error;
 }
 
+void Device::OnNewConnection(SecureSessionHandle session, SecureSessionMgr * mgr)
+{
+    mState         = ConnectionState::SecureConnected;
+    mSecureSession = session;
```

Comment:
Do we expect `SecureSessionHandle` to always be copyable?  Or are we likely to switch to move semantics on it at some point?  Put another way, should there be an `std::move()` here?

Reply:
Yes, I'll expect that the handle is POD, and always be copyable. In the future, only messaging layer will use the handle, the handle class won't be exposed to applications or controller API.